server/app/errorMiddleware/parseError.js

```diff
@@ -58,7 +59,7 @@ const parseNotValidate = e => {
 };
 
 const parseError = e => {
-  return e.type === 'Validation error' ? parseValidate(e) : parseNotValidate(e);
+  return e.type === 'Validation error' || 'unique violation' ? parseValidate(e) : parseNotValidate(e);
```

Comment:
The way I have the middleware structured, validations that are not in the 'validations' object (allowNull / unique) should be handled in parseNotValidate. I would add the code on lines 31 and 32 to line 55 after the if(). I think other models use 'name', so you might have to log the error object and find some other identifying keys.

Reply:
Right on. Removed the * || 'unique violation' * thing. I missed removing that.

I had it in there as I was figuring out the sequelize errors